Source/relay/TourGuide/Support/LocationAvailable.ash

```diff
@@ -868,10 +868,10 @@ static
         lookup_map["The Defiled Niche"] = "crypt.php";
         lookup_map["Haert of the Cyrpt"] = "crypt.php";
         lookup_map["Frat House"] = "island.php";
-        lookup_map["Frat House In Disguise"] = "island.php";
+        lookup_map["Frat House (frat disguise)"] = "island.php";
         lookup_map["The Frat House (Bombed Back to the Stone Age)"] = "island.php";
         lookup_map["Hippy Camp"] = "island.php";
-        lookup_map["Hippy Camp In Disguise"] = "island.php";
+        lookup_map["Hippy Camp (hippy disguise)"] = "island.php";
```

Comment:
same comment as before; it's Hippy Camp (in disguise) rather than Hippy Camp (hippy disguise). change at your leisure.

Reply:
is this with latest mafia? it was Hippy Camp (in disguise) in some earlier commits but Hippy Camp (hippy disguise) now